### DISCUSSION TITLE
The mingw version always produces NaN results when Out-Of-Core is enabled

### DISCUSSION THREAD

**Discussion by jfry207** [scivision/mumps-superbuild#64] [Q&A]
I compiled MUMPS with MinGW on windows (GCC 12.2.0)，It solves large scale problems correctly, but when I turn on OOC ICNTL(22)=1, the result is always Nan

BTW: The MSVC+OneAPI version works fine when i turn on OOC solving the same problem

**Comment by scivision** [comment]
I don't know if the MUMPS developers have tried this configuration. Is this using MPI (default is using MPI).   Or did you turn MPI off like:

```sh
cmake -Dparallel=off
```
